Makefile

```diff
@@ -13,18 +13,14 @@ xcodeproj-oss:
 
 bootstrap-common: check-dependencies common-crypto-mm postgres-mm ccmark-mm xcodeproj-mm init-db
 
-bootstrap-oss: mock-env mock-transcripts bootstrap-common xcodeproj-oss
+bootstrap-oss: mock-env bootstrap-common xcodeproj-oss
 
 bootstrap: submodules bootstrap-common xcodeproj
 
 mock-env:
 	test -f .env \
 		|| cp .env.example .env
 
-mock-transcripts:
-	test -d Sources/Server/Transcripts/ \
-		|| cp -r Transcripts.example/ Sources/Server/Transcripts/
```

Comment:
now that we have public episodes in this repo we don't even need mock transcripts. p cool